src/document/crdt/text.ts

```diff
@@ -179,7 +179,7 @@ export class CRDTText<A extends Indexable = Indexable> extends CRDTTextElement {
   /**
    * `create` a instance of Text.
    */
-  public static create<A = Indexable>(
+  public static create<A extends Indexable>(
```

Comment:
What is the intention of this change?

Reply:
The reason for changing to `extends Indexable` is that the generic type parameter A should be limited to the Indexable type or other types that inherit from Indexable.  (prevent ts2344 error)